jormungandr-lib/src/interfaces/block0_configuration/DOCUMENTED_EXAMPLE.yaml

```diff
@@ -55,11 +55,11 @@ blockchain_configuration:
   # total fees: constant + (num_inputs + num_outputs) * coefficient [+ certificate]
   linear_fees:
     # this is the minimum value to pay for every transaction
-    constant: 2
+    constant: 200000
     # the additional fee to pay for every inputs and outputs
-    coefficient: 1
+    coefficient: 100000
     # the additional fee to pay if the transaction embeds a certificate
-    certificate: 4
+    certificate: 400000
```

Comment:
I don't understand why you needed to change this value?

Reply:
See above.  This is 0.4 Ada.